eclipse-che/che-operator
# Switch to VS Code as the default Eclips Che editor

### What does this PR do?

Switch the default editor to VS Code. For existing instances the editor is not changed.

### Screenshot/screencast of this PR


### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/21771

### How to test this PR?

```bash
kubectl apply -f - <<EOF
apiVersion:  operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: eclipse-che-custom-catalog
  namespace: openshift-marketplace
spec:
  image: quay.io/mloriedo/che-operator-catalog:next
  sourceType:  grpc
  updateStrategy:
    registryPoll:
      interval: 5m
EOF

kubectl apply -f - <<EOF
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: eclipse-che-custom-sub
  namespace: openshift-operators
spec:
  channel: next
  installPlanApproval: Automatic
  name: eclipse-che-operator
  source: eclipse-che-custom-catalog
  sourceNamespace: openshift-marketplace
EOF
```

- create a CheCluster with defaults
- start a workspace without a `che-editor.yaml` file and verify that

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
